sdk/keyvault/azure-security-keyvault-keys/test/ut/key_client_base_test.hpp

```diff
@@ -122,4 +121,16 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Keys { nam
     }
   };
 
+  namespace {
+    static std::string GetSuffix(const testing::TestParamInfo<int>& info)
+    {
+      return std::to_string(info.param);
+    }
+  } // namespace
+
+  INSTANTIATE_TEST_SUITE_P(
```

Comment:
move this to cpp file, otherwise, this instantiation will happen every time this header is included. Looks like it is currently included in 7 cpp files, so we are producing 7 instances of each parametrized test.

Reply:
done